### ISSUE TITLE
Hello, thanks for your code!!! But I have some trouble using it

### ISSUE THREAD

**Issue by TonyDongLB** [milesial/Pytorch-UNet#23]
Hello, I’m doing a project about semantic segmentation with 2 class using your code, but my data was unbalanced, so I think maybe use CrossEntropyLoss with weight (just flat the output and label) is a good idea for my project, so changed the final layer from 1 channel to 2 channels,but the loss was unchanged during trainning, I’m so confused ,I give the label 0 or 1 to the mask, I don’t know where it is wrong. Thanks!

**Comment by milesial** [comment]
Hi, what parameters (batch size, number of epochs, training size, learning rate ...) are you using during training ? Is your loss slowly moving or not changing at all ?
